.vscode/launch.json

```diff
@@ -1,42 +1,29 @@
 {
     "version": "0.2.0",
     "configurations": [
-        {
-            // Use IntelliSense to find out which attributes exist for C# debugging
-            // Use hover for the description of the existing attributes
-            // For further information visit https://github.com/OmniSharp/omnisharp-vscode/blob/master/debugger-launchjson.md
-            "name": ".NET Core Launch (console)",
-            "type": "coreclr",
-            "request": "launch",
-            "preLaunchTask": "build",
-            // If you have changed target frameworks, make sure to update the program path.
-            "program": "${workspaceFolder}/csharp/tests/bin/Debug/net6.0/tests.dll",
-            "args": [],
-            "cwd": "${workspaceFolder}/csharp/tests",
-            // For more information about the 'console' field, see https://aka.ms/VSCode-CS-LaunchJson-Console
-            "console": "internalConsole",
-            "stopAtEntry": false
-        },
-        {
-            "name": ".NET Core Attach",
-            "type": "coreclr",
-            "request": "attach"
-        },
         {
             "name": "C# benchmark Launch (console)",
             "type": "coreclr",
             "request": "launch",
             "preLaunchTask": "build",
-            "program": "${workspaceFolder}/benchmarks/csharp/bin/Debug/net6.0/csharp_benchmark.dll",
-            "args": [],
+            "program": "${workspaceFolder}/benchmarks/csharp/bin/Debug/net8.0/csharp_benchmark.dll",
+            "args": [
+                "--dataSize",
```

Comment:
They are required parameters to start benchmark.

Reply:
nope, some of these arguments just fine-tune the benchmark. it can run without them - depending on the availability of a server.

Reply:
Ok. Removing. It wasn't working for me without args. So probably something was wrong on my side

Reply:
no worries. our benchmark documentation is lacking - it could be made more beginner-friendly. But from what I remember of the arguments you passed, all of them look completely optional. What errors did you receive?

Reply:
```csharp
public class CommandLineOptions
    {
        [Option('r', "resultsFile", Required = true, HelpText = "Set the file to which the JSON results are written.")]
        public string resultsFile { get; set; } = "";

        [Option('d', "dataSize", Required = true, HelpText = "The size of the sent data in bytes.")]
        public int dataSize { get; set; } = -1;

        [Option('c', "concurrentTasks", Required = true, HelpText = "The number of concurrent operations to perform.")]
        public IEnumerable<int> concurrentTasks { get; set; } = Enumerable.Empty<int>();

        [Option('l', "clients", Required = true, HelpText = "Which clients should run")]
        public string clientsToRun { get; set; } = "";

        [Option('h', "host", Required = true, HelpText = "What host to target")]
        public string host { get; set; } = "";

        [Option('C', "clientCount", Required = true, HelpText = "Number of clients to run concurrently")]
        public IEnumerable<int> clientCount { get; set; } = Enumerable.Empty<int>();

        [Option('t', "tls", Default = false, HelpText = "Should benchmark a TLS server")]
        public bool tls { get; set; } = false;


        [Option('m', "minimal", Default = false, HelpText = "Should use a minimal number of actions")]
        public bool minimal { get; set; } = false;
    }
```
They are marked as Required = true

Reply:
I can make all optional

Reply:
ah, shame on me - I thought about the install-and-test script. Of course, you're completely right - the benchmark requires the arguments.

Reply:
if you can make them optional (by using the defaults from the install-and-test script), that would be great.

Reply:
Unfortunately after checking with different wrappers  I found out (f.i. python) this fields also required. So I'm not sure it is the way to go.

Reply:
would you prefer that I open a PR for all wrappers?

Reply:
went and did it :)
https://github.com/aws/glide-for-redis/pull/898

Reply:
Cool!